src/utils/test.ts

```diff
@@ -49,3 +62,139 @@ export function withTestScheduler(
     }),
   );
 }
+
+export function mockMember(member: Partial<RoomMember>): RoomMember {
```

Comment:
We were discussing if we want to get those out of the js-sdk by moving it to an export.
I see how this would block this PR behind a js-sdk PR. Do you think a comment is a good idea to clarify that there might be a good chance to share code with the js-sdk here?